src/server/ua_services_attribute.c

```diff
@@ -411,9 +411,7 @@ void Service_Read(UA_Server *server, UA_Session *session, const UA_ReadRequest *
 		size_t offset = 0;
         UA_Connection *c = UA_NULL;
         UA_SecureChannel *sc = session->channel;
-        if(sc)
```

Comment:
The problem is that I am not sure myself ;) Can you just, revert this line and mark it with "TODO" or "FIXME" and/or open an issue